docs/tutorials/tagged-releases.md

```diff
@@ -0,0 +1,92 @@
+# Creating Tagged Releases
+
+This tutorial covers creating Tagged Releases with Github, Travis and Mission Control.
+
+The ability to create tagged releases in our molo sites, allows us to test a QA version of our code, without merging our changes into the develop branch. Note that this currently only works for sites like TuneMe and Springster, not molo.core or a particular package.
+
+This tutorial will explain how to create a tagged release for a branch on TuneMe, as an example.
+
+## Creating your first tagged release
+
+### Ensure everything is configured
+
+Make sure that Travis is set up for to create tagged releases. See [this commit](https://github.com/praekelt/molo-tuneme/pull/489/commits/5eb3488433b0c8daba2fd37f6f9a883c2f446d31) as an example of how to do this.
+Check with SRE that the account is set up to handle tagged releases on DockerHub. At the time of writing this, both Springster and TuneMe are set up to handle tagged releases.
```

Comment:
Docker Hub

Reply:
I also don't know that SRE would be the best people to ask. I set this up, but not sure anybody else knows about this. I'm not sure if there actually is much to check-- if the account can push to Docker Hub then it can push tagged images too.